Makefile

```diff
@@ -75,6 +77,8 @@ uninstall:
 
 clean: purge uninstall ## Make a clean source tree and uninstall extensions
 
+full-clean: purge purge-yarn uninstall ## Should be unnecessary to run this
```

Comment:
clean should be the `full clean`. Why creating a new target? Just add purge-yarn to clean.